app/views/registers/show.js.erb

```diff
@@ -1,8 +1,10 @@
-$("#records-tbody").append("<%= j(render(partial: 'record', collection: @records)) %>");
-$("#records-count").html('Showing <strong>1 &ndash; <%= @records.count + @records.offset_value %></strong> of <strong><%= @records.total_count %></strong> records <% if @search_term.present? %>for <strong>"<%= search_term %>"</strong><% end %>')
+document.getElementById('records-tbody').innerHTML += "<%= j(render(partial: 'record', collection: @records)) %>";
+
+document.getElementById('records-count').innerHTML = 'Showing <strong>1 &ndash; <%= @records.count + @records.offset_value %></strong> of <strong><%= @records.total_count %></strong> records <% if @search_term.present? %>for &ldquo;<strong><%= @search_term %></strong>&rdquo;<% end %>';
```

Comment:
should we use `number_with_delimiter` here too for consistency?

Reply:
Oops. Now committed.